cmd/cnab-run/install.go

```diff
@@ -84,3 +86,14 @@ func getBundleImageMap() (map[string]bundle.Image, error) {
 	}
 	return result, nil
 }
+
+func addAppLabels(rendered *composetypes.Config, instanceName string) {
+	for i, service := range rendered.Services {
+		if service.Labels == nil {
+			service.Labels = map[string]string{}
+		}
+		service.Labels["com.docker.app.namespace"] = instanceName
```

Comment:
Can we create constants for the label keys (they are used in two places)?

Reply:
Where would be the best place to put these consts? Just at the top of this file alongside `imageMapFilePath`?
Also, should the consts be public (or whatever the term in go is)?

Reply:
You can put them in `names.go`. They should be public because they are used in another package